package.json

```diff
@@ -21,47 +21,44 @@
   },
   "dependencies": {
     "ember-cli-babel": "^5.1.7",
-    "ember-getowner-polyfill": "^1.2.2"
+    "ember-getowner-polyfill": "^1.1.1"
```

Comment:
We can actually work with 1.1.1 as well and to allow for better deduplication with yarn I think we should specify the dependency version as relaxed as possible.